website/_config.yml

```
@@ -0,0 +1,25 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+title: Carbon language documentation
```

While I understand you wanted to follow the guidance on capitalization here, I think this should still capitalize `Language` as that's part of the proper name "Carbon Language"?

(We also capitalize it in other places such as the title text of the main readme.)

```suggestion
title: Carbon Language documentation
```